src/main/menu/templates/paragraph.js

```diff
@@ -56,14 +56,14 @@ export default function (keybindings) {
       type: 'separator'
     }, {
       id: 'upgradeHeadingMenuItem',
-      label: 'Upgrade Heading',
+      label: 'Promote Heading',
```

Comment:
These verbs apply to numbers and quantities, such as header level, not a header itself. Other verbs could be considered instead of "upgrade" and "demote", for example, "raise" and "lower", but these are more confusing because of the implication of movement up or down in the document position.

Reply:
In general, we increase the heading level but promote is fine, as we want to keep it simple.

Reply:
I think the best options are "Increase Heading Level" and "Promote Heading". The trade off is a more common word versus a shorter phrase. If icons are added eventually then they would help to add understanding from the word.

Reply:
It's up to you, both are fine.